src/components/Dialog.jsx

```diff
@@ -35,7 +35,7 @@ class Dialog extends React.Component {
   }
 
   render() {
-    const zInd = this.props.component === this.props.focusedDialog ? '100' : '0';
+    const zInd = this.props.component === this.props.focusedDialog ? '100' : '5';
```

Comment:
I don't see the difference between z-index 0 and z-index 5 for a Dialog that doesn't have focus; I see that both values work more or less the same (i.e. the "non-focused" dialog appears behind the dialog with focus) given the scenarios I'm testing on OSX+Chrome. Can you tell me what I'm missing, or what this was intended to fix?

(Are there browsers where z-index 0 meant the non-focused dialog disappears into the page behind it?)